### ISSUE TITLE
Could not login frontend if REST API is added by Studio 14

### ISSUE THREAD

**Issue by IskandarovRost** [cuba-platform/frontend#199]
See https://youtrack.cuba-platform.com/issue/STUDIO-7098. After this feature was implemented could not login Frontend client.  

1) Using Studio **14** create frontend module 
![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/dfd5e661-220d-45d1-8fb6-618cd73b7ce5)

2) Allow REST API for Admin (add a 'rest api'-scope role with the corresponding specific permissions
![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/da375036-22e0-4089-ad07-8e26d9da6200)

3) start the application 

4) try to login frontent client with the Admin user

AR: Login failed. 
```
curl -H "Content-type: application/x-www-form-urlencoded" -H "Authorization: Basic Y2xpZW50OnNlY3JldA==" -d "grant_type=password&username=admin&password=admin" http://localhost:8080/app/rest/v2/oauth/token

{"error":"unauthorized","error_description":"Bad credentials"}
```

![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/ead84ef4-2715-4d32-b88b-464451e9a927)

**Comment by vadimbasko** [comment]
Fixed as part of #222

**Comment by IskandarovRost** [comment]
Verified.
But STUDIO-8721 appeared
